src/main/java/org/springframework/data/elasticsearch/core/ElasticsearchRestTemplate.java

```diff
@@ -147,6 +147,8 @@
 		implements ElasticsearchOperations, EsClient<RestHighLevelClient>, ApplicationContextAware {
 
 	private static final Logger logger = LoggerFactory.getLogger(ElasticsearchRestTemplate.class);
+	private static final String FIELD_SCORE = "_score";
```

Comment:
_FIELD_SCORE_ also exists in `ElasticsearchTemplate`. Please move it to `AbstractElasticsearchTemplate`

Reply:
I will use the public constant `ScoreSortBuilder.NAME` instead in both `ElasticsearchRestTemplate` and `ElasticsearchTemplate`.